### PR TITLE
Minor Sidebar Navigation and Layout Changes

### PR DESCRIPTION
**Pull request by Locrian24**
- Changed location of the "return" button from an arrow to button on top of list
- Added more FA icons to dev center
- Fixed issue with sidebar over-extending when screen width is reduced
- Integrations page only on main, home section
- Fixed alignments of elements when paired with FA icons

### CODE REVIEW

**Comment by loganvolkers** [comment]
Thanks for the pull request @Locrian24!

> - Fixed alignments of elements when paired with FA icons

**This looks really nice!** I'll likely pluck this change onto production today. We just launched up upgrade from Metalsmith to React-Static, so unlikely we'll be able to merge this directly without you doing a rebase.

> - Changed location of the "return" button from an arrow to button on top of list

I think this hides the button too much.

> - Added more FA icons to dev center

The lack of icons was an intentional choice, it helps highlight the ones with icons.

> - Fixed issue with sidebar over-extending when screen width is reduced

I'll have to see what you're talking about here.

> - Integrations page only on main, home section 

I'll see what @rebeccachan thinks

P.S. You can see your changes in Netlify here: https://deploy-preview-87--saasquatch-docs.netlify.com/developer/

File: src/templates/sidebar.html [hunk 1]
```diff
@@ -1,302 +1,525 @@
...
+          >
+        </li>
+        <li>
+          <a href="/topics/json-web-tokens"
+            ><i class="fa small fa-fw fa-lock"></i> JSON Web Tokens</a
+          >
+        </li>
+        <li>
+          <a href="/developer/testing"
+            ><i class="fa fa-fw small fa-file"></i> Testing Best Practices</a
```
Comment (loganvolkers) [inline src/templates/sidebar.html]: More iconic for testing.

```suggestion
            ><i class="fa fa-fw small fa-flask"></i> Testing Best Practices</a
```

File: src/templates/sidebar.html [hunk 2]
```diff
@@ -1,302 +1,525 @@
...
+            </li>
+            <li><a href="/api/authentication">Authentication</a></li>
+            <li><a href="/api/openendpoints">API Open Endpoints</a></li>
+            <li><a href="/api/errors">Errors</a></li>
+            <li><a href="/api/webhooks">Webhooks</a></li>
 
+            <li class="Divider">API Reference</li>
+            <li><a href="/api/methods">Full list of Methods</a></li>
 
+            <APILIST />
```
Comment (loganvolkers) [inline src/templates/sidebar.html]: The space breaks this.

```suggestion
            <APILIST/>
```

**Comment by rebeccachan** [comment]
> Fixed alignments of elements when paired with FA icons
> This looks really nice! I'll likely pluck this change onto production today. We just launched up upgrade from Metalsmith to React-Static, so unlikely we'll be able to merge this directly without you doing a rebase.

Sweet! The only thing is the main "Integrations" page on the splash should stay larger. 

> Changed location of the "return" button from an arrow to button on top of list
> I think this hides the button too much.

I actually quite like this, as many people missed the arrow at the top and were just clicking the logo to return to home. Maybe both would be good?

> Added more FA icons to dev center
>The lack of icons was an intentional choice, it helps highlight the ones with icons. 

@loganvolkers Agreed, please keep the original version with fewer icons.

> Fixed issue with sidebar over-extending when screen width is reduced
> I'll have to see what you're talking about here.
 
@loganvolkers Not sure where this is happening, as I don't see it on the deploy but I only tested it on Chrome.

> Integrations page only on main, home section

Integrations also exists in the Dev center in live deploy, must have been removed accidentally in this edit :)

File: src/templates/sidebar.html (hunk 2 above)
Comment (Locrian24) [reply src/templates/sidebar.html]: This seems to have been from Prettier's auto-formatting. Will have to use a comment just to make sure it ignores this line

**Comment by Locrian24** [comment]
@loganvolkers @rebeccachan Thank you for review so far! I've committed edits to my fork based on your comments. Changes made are listed below:

- Icons in Dev Center are back to the original
- Integration page link back to being also in Dev and Success Center
- Integration link on splash back to increased font
- Both navigation buttons are now visible

> Fixed issue with sidebar over-extending when screen width is reduced
> I'll have to see what you're talking about here.
> @loganvolkers Not sure where this is happening, as I don't see it on the deploy but I only tested it on Chrome.

Seems to have only been a Firefox issue. Reproducible on the live version in Firefox by gradually decreasing screen width in the browser while having the sidebar expanded.

**Review by loganvolkers — APPROVED** [review]
Looks good, thanks @Locrian24!

**Pull request merged**